nx-dev/ui-cloud/src/lib/faster-and-cheaper.tsx

```diff
@@ -0,0 +1,165 @@
+'use client';
+import {
+  Variants,
+  animate,
+  motion,
+  useMotionValue,
+  useTransform,
+} from 'framer-motion';
+import { useEffect } from 'react';
+import { Spotlight } from './elements/spotlight';
+
+export function FasterAndCheaper(): JSX.Element {
+  const spotlight: Variants = {
+    offscreen: {
+      display: 'none',
+    },
+    onscreen: {
+      display: 'block',
+    },
+  };
+
+  const ciBar: Variants = {
+    start: {
+      width: 0,
+    },
+    end: {
+      width: '100%',
+      transition: {
+        type: 'tween',
+        ease: 'easeOut',
+        duration: 1.2,
+      },
+    },
+  };
+  const nxBar: Variants = {
+    start: {
+      width: 0,
+    },
+    end: {
+      width: '50%',
+      transition: {
+        type: 'tween',
+        ease: 'easeOut',
+        duration: 1.2,
+      },
+    },
+  };
```

Comment:
Can you remove the animation altogether on the bars? This will stagger the mobile version otherwise.